ocs_ci/ocs/constants.py

```diff
@@ -1602,7 +1603,7 @@
 # Miscellaneous
 NOOBAA_OPERATOR_POD_CLI_PATH = "/usr/local/bin/noobaa-operator"
 NOOBAA_OPERATOR_LOCAL_CLI_PATH = os.path.join(DATA_DIR, "mcg-cli")
-CLI_TOOL_LOCAL_PATH = os.path.join(DATA_DIR, "odf-cli")
+CLI_TOOL_LOCAL_PATH = os.path.join(BIN_DIR, "odf-cli")
```

Comment:
you are not using it anywhere in this PR right?
But I see references in ocs_ci/helpers/helpers.py . 

So please change it there, to use directly path of bin dir (config.RUN['bin_dir']) from config we talked yesterday.

As it will be in the path, I think that you will really need anywhere else the full path as you will be able to just run directly `odf-cli`